components/abstractions/src/main/java/com/microsoft/kiota/serialization/SerializationWriterProxyFactory.java

```diff
@@ -13,7 +13,7 @@ public abstract class SerializationWriterProxyFactory implements SerializationWr
         return _concrete.getValidContentType();
     }
 
-    private final SerializationWriterFactory _concrete;
+    protected final SerializationWriterFactory _concrete;
```

Comment:
We should update the field name to follow naming conventions since it's not private anymore.